libraries/tock-register-interface/src/lib.rs

```diff
@@ -61,6 +61,7 @@
 // If we don't build any actual register types, we don't need unsafe
 // code in this crate
 #![cfg_attr(not(feature = "register_types"), forbid(unsafe_code))]
+#![feature(doc_cfg)]
```

Comment:
This might be an issue.

Reply:
Ah ok, I'll remove it. it's just for fancy docs. I'll remove it then.
I'll just make 'debug' be generated for doc so it's clear for the user.

Why it's an issue btw? Curious

Reply:
Right, we're trying to reduce the number of nightly features we depend on: #1654.

Reply:
Indeed, I removed it.